.travis.yml

```diff
@@ -29,17 +29,17 @@ env:
       COVERAGE=true
```

Comment:
This build fails because Ubuntu Precise only has numpy 1.6.1 - should Travis builds switch to `dist: trusty` now?